diff -up libgdiplus-2.10.9/configure.in.automake-1_13 libgdiplus-2.10.9/configure.in
--- libgdiplus-2.10.9/configure.in.automake-1_13	2013-01-08 02:04:14.993233123 +0100
+++ libgdiplus-2.10.9/configure.in	2013-01-08 02:07:09.731568146 +0100
@@ -1,6 +1,6 @@
 AC_INIT(README)
 AC_CANONICAL_SYSTEM
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AM_INIT_AUTOMAKE(libgdiplus,2.10.9)
 
 AM_MAINTAINER_MODE
